src/go.sum

```diff
@@ -1,5 +1,7 @@
 github.com/kisom/netallow v0.0.0-20200609175051-08f6b004e41a h1:4T7cUpk4OIqaxn7i41yVYEU7/4gjTKuvqlrSqbBwwe0=
 github.com/kisom/netallow v0.0.0-20200609175051-08f6b004e41a/go.mod h1:fHrMiR3Isu09r3MBg9oqlp0E+qBtRp6qsig0hpmgXYg=
+github.com/robfig/cron v1.2.0 h1:ZjScXvvxeQ63Dbyxy76Fj3AT3Ut0aKsyd2/tl3DTMuQ=
```

Comment:
It looks like upstream has published robfig/cron v3: <https://github.com/robfig/cron>

Are there any reasons to stick to v1.2?

Reply:
I just `go get` without specific version and it gets v1.2. I will upgrade it to v3 later.

Reply:
Upgrade at new commit.